istresearch/rp-indexer
# P4-2034 - Upstream 5.6.0 merge

# For the Reviewer

When this is complete, you should approve the PR via github.

## Summary
Upstream 5.6.0 merge, no changes. 

#### Release Note
Merged upstream 5.6.0. 

#### Breaking Changes
None.

## Quality Assurance

You have gathered the following items:

Given all of the items above, you have updated your Application ORT at the following locations:
- **Features and Alerting**: <link to app ORT>Required.
- **P4 Alerting**: <link to p4-alerting PR>Required.

## Testing and Verification

There are no functional changes in this PR.

To test:

1. Spin up an engage 4.0.5. dev stack, using the `istresearch/rp-indexer:ci-4.0.5-dev-P4-2034` image (this will work on 4.0.4 dev also) 
2. Ensure you can search for contacts within the UI, including newly created contacts.
